src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,5 +4,17 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
-        return String.valueOf(n).contains("2");
+        int c=2;
+        for (int i = 1; i < c; i++) {
+            int find = n % 10;
+            if (find == 2) return true;
+            if ((n / 10) == 0) break;
+            n /= 10;
+            c += 1;
```

Comment:
Will you be able to explain following solution ?
As for me, it's too complicated and have quite weird logic.